flink-end-to-end-tests/test-scripts/docker-mesos-cluster/docker-compose.yml

```diff
@@ -53,7 +53,6 @@ services:
     networks:
       - docker-mesos-cluster-network
     volumes:
-      - /var/run/docker.sock:/var/run/docker.sock
       - ${END_TO_END_DIR}:${END_TO_END_DIR}
```

Comment:
We need to verify the hash of result. So, I mount the output dir in container. Otherwise, I could not found a way to get the result from the sandbox of Mesos.

Reply:
I suggest to use `docker cp` to get the output to local and then verify. Using docker volume and mount may cause permission problems when cleanup logs and output.

Reply:
I think we need to avoid `privileged container` as much as possible too. Since it may cause the same problem.